extensions/wikia/UserPreferencesV2/UserPreferencesV2.i18n.php

```diff
@@ -15,19 +15,18 @@
 	'preferences-v2-date' => 'Date format:',
 	'preferences-v2-time' => 'Time zone:',
 	'preferences-v2-skin' => 'Layout:',
-
 	'prefs-emailv2' => 'Email',
 	'prefs-email-me-v2' => 'Email me when...',
 	'tog-enotifwatchlistpages-v2' => "...a page I'm following is changed",
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "tog-enotifwatchlistpages-v2" 27 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)